tests/predictor/test_torch_stoi.py

```diff
@@ -0,0 +1,178 @@
+"""Module for testing torch_stoi.py"""
+import numpy as np
+import pytest
+import torch
+from pystoi.stoi import FS
+from torch.testing import assert_close
+
+from clarity.predictor.torch_stoi import (
+    NegSTOILoss,
+    masked_mean,
+    masked_norm,
+    meanvar_norm,
+)
+
+# pylint: disable=redefined-outer-name,unused-argument  # pytest fixtures
```

Comment:
These `pylint` errors are caused by having `pytest.fixtures` defined within the file as noted.

They can be avoided by placing fixtures in `tests/conftest.py` if desired.

Reply:
Thanks.  That's useful to know -- didn't realise conftests could be defined for each directory. Only problem with that is that it still defines the fixtures at the directory level which means that in some cases imports get pulled in for tests that don't need them. The real problem here I think is that pylint-pytest wasn't installed in my environment -- pylint-pytest is designed to fix some of these pytest-related false positives. I've now managed to check in ok without needing these disables.